### ISSUE TITLE
Print font too small

### ISSUE THREAD

**Issue by refactoringdr** [504rules/504rules.github.io#121]
When printing the rules, it uses a 10px font, which is almost unreadable to my old eyes. I propose changing it to 14px, which is readable to me. Any comments or complaints? I'll leave this open for a few days, and if no complaints, I'll change it.

**Comment by SteveOliverC** [comment]
14 pt seems too large. As it is now, many of the games will fit onto a single page which is nice. So going to 14 pt will require another page for many games.

Unless there is an way to let the user select the font size they want to print in?

**Comment by refactoringdr** [comment]
Right now, it is 10px (not pt). I’ve attached a .pdf of what prints on my system. There’s not an easy way to select the font size, since it is defined in .css. I understand the value of fitting onto a single page (most games still do). But since it’s unreadable to >40 yr olds, I would end up having to use an online version. I know 987 will overflow onto a third page. I think many of the 9 worlds would.

don

**Comment by Arachnid** [comment]
Are there any tweaks we can make to formatting to make games still fit on a single page even with a larger font size, such as reducing margins or header spacing?

**Comment by SteveOliverC** [comment]
I'm >50 and I can read it. Let's not say that it's "unreadable" to everyone >40 at the current print size.

**Comment by refactoringdr** [comment]
Fair enough. At 12px, 987 fits onto 2 pages. Do you know of a longer game word of the top of your head?

don

**Comment by refactoringdr** [comment]
I guess I’ll give up. 913 takes up 3 pages at even 12px.

don

**Comment by Arachnid** [comment]
I'd suggest adjusting font size down for particularly long games, but detecting that is... nontrivial in CSS & JS.

**Comment by refactoringdr** [comment]
Actually, at 12px, it looks likes I can get 913 on 2 pages. What is the world with the longest ruleset?

don

**Comment by refactoringdr** [comment]
914 fits on 2 pages at 12px. Again, does anyone know of longer ruleset?

**Comment by Shalavar** [comment]
297 is readable but fuzzy. Using a Ricoh laser printer. Definitely not translating from digital to printed very well at all.
